sdk/servicebus/service-bus/migrationguide.md

```diff
@@ -33,8 +33,19 @@ We now have methods with similar names, signature and location to create senders
 We have a variety of new features in the version 7 of the Service Bus library.
 
 - A new `ServiceBusAdministrationClient` to perform operations like create/get/list/update/delete on queues/topics/subscriptions/rules. These were already available as part of a separate package `@azure/arm-servicebus` that uses Azure Resource Manager APIs but had the drawback of not supporting connection strings.
+- Ability to create a batch of messages with the smarter `ServiceBusSender.createBatch()` and `ServiceBusMessageBatch.tryAddMessage()` APIs. This will help you manage the messages to be sent in the most optimal way.
 - Ability to configure the retry policy used by the operations on the client, sender and receivers.
-- Ability to cancel async operations on the client, sender and receivers using the abort signal from `@azure/abort-controller`
+- Ability to cancel async operations on the client, sender and receivers and the management operations using the abort signal from `@azure/abort-controller`.
+- Ability to perform tracing with `[@azure/core-tracing](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-tracing/README.md)`
+- Authentication with AAD credentials using `@azure/identity`.
+  Refer [Changelog.md](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/CHANGELOG.md) for more new features, changes and bug fixes.
+
+### Performance Improvements
+
+Notable performance improvements.
+
+- Number of messages that can be sent in a certain duration using a single sender has been improved by about 4-5 folds(excluding the effects of batch message API).
+- Proper cleanup and better listener management affecting workflows in multiple areas such as message lock renewals, session lock renewals, peeking into messages would improve the memory usage and avoids potential memory leaks.
```

Comment:
```suggestion
- Memory usage for message lock renewals, session lock renewals and peeking has been made more efficient.
```

(maybe? I'm not sure if we have to mention listeners here)

Reply:
Hmmm.

Reply:
I've taken in your changes, changelog mentions detailed information, this might be enough for here then.